examples/lib/commons/square_component.dart

```diff
@@ -2,21 +2,18 @@ import 'dart:ui';
 
 import 'package:flame/components.dart';
 import 'package:flame/effects.dart';
-import 'package:flame/palette.dart';
+import 'package:flutter/material.dart';
```

Comment:
Just curious why we importing material here?

Reply:
No idea... Why didn't our linter complain at that?
Removed it now.